README.md

```diff
@@ -16,20 +16,20 @@ Once these are installed, you can configurable and build this project using
 cmake. To do so, the user should manually set the following two variables:
 
 1. `CABANA_DIR` (pointing to the install location of Cabana)
```

Comment:
Do you still need this? Shouldn't https://github.com/ECP-copa/Cabana/pull/165 help to use `CMAKE_PREFIX_PATH` here as well?